src/Attestation/Attestation/help/Get-AzAttestationPolicySigners.md

```diff
@@ -0,0 +1,145 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Attestation.dll-Help.xml
+Module Name: Az.Attestation
+online version:https://docs.microsoft.com/en-us/powershell/module/az.attestation/get-azattestationpolicysigners
+schema: 2.0.0
+---
+
+# Get-AzAttestationPolicySigners
+
+## SYNOPSIS
+Gets the trusted policy signers from a tenant in Azure Attestation.
+
+## SYNTAX
+
+### NameParameterSet
+```
+Get-AzAttestationPolicySigners [-Name] <String> [-ResourceGroupName] <String>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ResourceIdParameterSet
+```
+Get-AzAttestationPolicySigners [-ResourceId] <String> [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The Get-AzAttestationPolicySigners cmdlet gets the trusted policy signers from a tenant in Azure Attestation.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-AzAttestationPolicySigners -Name "myservice" -ResourceGroupName "myrg"
+```
+
+Gets the trusted policy signers for the tenant named "myservice".
+
+## PARAMETERS
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.Core.IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzContext, AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Name
+Specifies the name of an attestation provider.
+
+```yaml
+Type: System.String
+Parameter Sets: NameParameterSet
+Aliases:
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ResourceGroupName
+Specifies the resource group name of an attestation provider.
+
+```yaml
+Type: System.String
+Parameter Sets: NameParameterSet
+Aliases:
+
+Required: True
+Position: 1
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ResourceId
+Specifies the ResourceID of an attestation provider.
+
+```yaml
+Type: System.String
+Parameter Sets: ResourceIdParameterSet
+Aliases:
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -Confirm
```

Comment:
this comes from SupportShouldProcess I believe

Reply:
Thanks, fixed.